express-middlewares/expressMiddlewares.js

```diff
@@ -1,8 +1,34 @@
+const express = require('express');
 
-function expressMiddlewares(...args) {
-  return args;
-}
+const app = express();
 
-export {
-  expressMiddlewares,
-};
+app.use((req, res, next) => {
+  const url = req.originalUrl.slice(1);
+  res.setHeader('middleware-header', url);
+  console.log(`Some user visited ${url} at ${new Date()}`);
```

Comment:
Should you require the text `Some user visited` ?
I think `Time /path` is good to go.